functors_task_1/student/include/impl.h

```diff
@@ -8,6 +8,19 @@
  */
 struct BeerOrganizer
 {
+    BeerOrganizer() : mBeerBrand{BeerBrand::None} {}
```

Comment:
Use in place initialization, avoid constructors
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c45-dont-define-a-default-constructor-that-only-initializes-data-members-use-in-class-member-initializers-instead